pom.xml

```diff
@@ -160,12 +160,12 @@
 		<dependency>
 			<groupId>com.force.api</groupId>
 			<artifactId>force-partner-api</artifactId>
-			<version>34.0</version>
+			<version>36.0.0</version>
 		</dependency>
 		<dependency>
 			<groupId>com.force.api</groupId>
 			<artifactId>force-wsc</artifactId>
-			<version>34.0</version>
+			<version>36.1.1</version>
```

Comment:
yes they both need to maintain parity regardless of scope of separation of those binaries